### DISCUSSION TITLE
My bot breaks when other bots send messages.

### DISCUSSION THREAD

**Discussion by rubot22** [python-telegram-bot/python-telegram-bot#4433] [Q&A]
I am making a bot for a telegram groupchat I am in, and this groupchat has two other bots in it already, namely the "GroupHelpBot" and "RollEmBot".

Whenever someone sends a command to one of the other two bots, my bot will stop functioning. Restarting the bot doesn't fix it, and I have to resort to replacing the Telegram API key entirely to get things working again. No error shows up in the console either.

I tried a variety of handlers, and even made a custom one in an attempt to catch all update types. Nothing works and the issue is frustrating because I have to make a new bot, add it to the chat, and update the API key in my code every time the issue occurs. I am not sure what to do about it.
```
class CatchAllHandler(BaseHandler[Update, CCT]):
    __slots__ = ()

    def check_update(self, update: object) -> bool:
        return isinstance(update, Update) and bool(update.ALL_TYPES)

**Comment by Poolitzer** [comment]
Hi, if you expect to continue getting updates, you need to disable privacy mode or add the bot as admin.

However commands send to your bot via the command menu in the group chat should result in an update no matter what.
